### DISCUSSION TITLE
Unauthorized access. No direct access allowed to this module.

### DISCUSSION THREAD

**Discussion by universowebstore** [sensson/whmcs-incasso#11] [Q&A]
I cant manage my subscription batches, help pls, im with the admin account on my WHCMS

**Comment by ju5t** [comment]
@universowebstore can you share some more details?

- What's your PHP version?
- Which version of WHMCS are you using?
- Which browser are you using?
- What changes have been made recently (like the last 2/3 weeks)?

Can you also try the following:

- In WHMCS go to the direct debit module and select the manage subscription batch tab.
- Open your inspection window, on Chrome you can get there with View > Developer > Developer Tools, or alternatively hold option+command+i if you're on MacOS.
- Go to the Network tab and select Fetch/XHR.
- Select a different subscription batch. You should see something happening. For example:

![CleanShot 2024-04-26 at 16 26 46](https://github.com/sensson/whmcs-incasso/assets/3635751/14ee6e26-f94f-4427-b721-d2595c4610cc)

I don't need to know what it says, all I need to know is if you see your configured hash after `hash=` in that url. This hash is configured under subscription settings.

**Comment by universowebstore** [comment]
PHP 8.1.28
WHCMS 8.8.0
Chrome 
Update DonDominio module

**Comment by universowebstore** [comment]
i only have 1 batch and it says Unauthorized access. No direct access allowed to this module.

**Comment by universowebstore** [comment]
now its working, thx
